dictionaries/companies/src/companies.txt

```diff
@@ -933,6 +934,7 @@ Johnson Controls, Inc.
 Joy Global Inc.
 Joyent
 JPMorgan Chase & Co.
+jsdelivr
```

Comment:
This is more of a [web services](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/software-services.txt) than a company. Even the founder uses another company name for himself: [founder: `dak@prospectone.io`](https://www.jsdelivr.com/about#people)